### ISSUE TITLE
Fields not labelled correctly

### ISSUE THREAD

**Issue by mkaluzny** [beemailer/bee-field#1]
The purpose of `for` attribute of `<label>` tag is to provide an **`id`** of the labeled field, and none of the fileds generated by bee-field has an id. Currently click on a label has no effect.
Maybe it will be a good idea to add an option to provide a field-id as directive attribute, and if it is not provided the field id will be set to the same value as the name attribute?

**Comment by beemailer** [comment]
yes you're right and that should be an easy fix, feel free to submit a PR or I'll implement it as soon as I can
